client/src/extension.ts

```diff
@@ -97,6 +100,8 @@ export function activate(context: ExtensionContext) {
  * @param commandName The Yseop CLI subcommand to use, like ”batch” or ”test”.
  */
 export async function ExecYseopCliCommand(yseopCliPath: string, ...words: string[]) {
+	yseopCliOutputChannel.clear();

```

Comment:
Isn't clearing a bit brutal? People sometimes like to see the results of all previous actions.
You're more accustomed to VSCode than I am, so I'll let you take the final decision I guess.

Reply:
We could have left an empty line, but since our output is still a bit messy, it would still be kinda hard to read. Kay.